init.go

```diff
@@ -112,6 +112,26 @@ func initFn() cli.Command {
 }
 
 func (a *initFnCmd) init(c *cli.Context) error {
+	wd := getWd()
+
+	path := c.Args().First()
+	if path != "" {
+		fmt.Printf("Creating function at: /%s\n", path)
+		dir := filepath.Join(wd, path)
+		// check if dir exists, if it does, then we can't create function
+		if exists(dir) {
+			return fmt.Errorf("directory %s already exists, cannot init function", dir)
```

Comment:
Let users decide fail or not, if folder exist - allow user to override whole structure (`--force` maybe?)

Reply:
Fixed.